packages/edit-post/src/components/welcome-guide-modal/index.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * WordPress dependencies
+ */
+import { useSelect, useDispatch } from '@wordpress/data';
+import { Modal } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { __experimentalCreateInterpolateElement } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import Guide from './guide';
+import { CanvasImage, EditorImage, BlockLibraryImage, InserterIconImage } from './vectors';
+
+export default function WelcomeGuideModal() {
+	const areTipsEnabled = useSelect( ( select ) => select( 'core/nux' ).areTipsEnabled() );
+
+	const { disableTips } = useDispatch( 'core/nux' );
+
+	if ( ! areTipsEnabled ) {
+		return null;
+	}
+
+	return (
+		<Modal
+			className="edit-post-welcome-guide-modal"
+			shouldCloseOnClickOutside={ false }
```

Comment:
I've disabled this to work around an issue where the modal is immediately dismissed on page load because the post's title field receives focus. Let me know if you can think of a better alternative!